lib/patch_status_json.py

```diff
@@ -47,8 +47,9 @@ def patch_status_json_schema_0_1(status_file: str):
         "name": new_status.pop("subscription", ""),
         "id": new_status.pop("subscription-id", ""),
     }
-    new_status["execution_status"] = new_status.pop("configStatus")
-    new_status["execution_details"] = new_status.pop("configStatusDetails")
+    if "configStatus" in new_status:
+        new_status["execution_status"] = new_status.pop("configStatus")
+        new_status["execution_details"] = new_status.pop("configStatusDetails")
```

Comment:
Just to be extra safe here, I think we need to make the same check for `configStatusDetails`

Reply:
Both were introduced at the same point, so one is present only when the other is too.
But I can add it just to be safe if you think it's needed

Reply:
Yep, I do realize that. It is just to be extra safe, but maybe it is indeed not needed

Reply:
too late, added hahaha